tools/console/console.js

```diff
@@ -1180,13 +1183,9 @@ class Console extends ConsoleBase {
     } else if ((! this._stream.isTTY) || (! this._pretty)) {
       // No progress bar if not in pretty / on TTY.
       newProgressDisplay = new ProgressDisplayNone(this);
-    } else if (this._stream.isTTY && ! this._stream.columns) {
-      // We might be in a pseudo-TTY that doesn't support
-      // clearLine() and cursorTo(...).
-      // It's important that we only enter status message mode
-      // if this._pretty, so that we don't start displaying
-      // status messages too soon.
```

Comment:
I removed this as it seemed a bit verbose, but in hindsight, I guess this comment was referring to the importance of this being later on in the conditional chain (after the `else if` on 1180).  Happy to add it back if its removal detracts from the readability.

Reply:
I'm okay with leaving it out! 👍